blocks/media-upload/index.js

```diff
@@ -19,7 +19,7 @@ const getGalleryDetailsMediaFrame = () => {
 		/**
 		 * Create the default states.
 		 *
-		 * @returns {void}
+		 * @returns {void} Void.
```

Comment:
This is not necessary. The `{void}` return type removes the need for a return description.